### ISSUE TITLE
Custom Logo not working

### ISSUE THREAD

**Issue by swoopapp** [marmelab/react-admin#2975]
**What you were expecting:**
I was expecting to see my custom LOGO in the app bar

**What happened instead:**
I dont see the bug in the header

**Steps to reproduce:**
I have copied the steps in the readme to include a custom logo with the customer app bar

**Related code:**

```
insert short code snippets here
import React from "react";
import Typography from "@material-ui/core/Typography";
// import AppBar from "@material-ui/core/AppBar";
import { AppBar } from "react-admin";
import Toolbar from "@material-ui/core/Toolbar";
import { withStyles } from "@material-ui/core/styles";

import Logo from "./Logo";

const styles = {
  title: {
    flex: 1,
    textOverflow: "ellipsis",
    whiteSpace: "nowrap",
    overflow: "hidden"
    // color: "black",
    // backgroundColor: "white",
    // height: "80px"
  },
  spacer: {
    flex: 1
  }
};

const CustomAppBar = withStyles(styles)(({ classes, ...props }) => (
  <AppBar
    //   className={classes.title}
    {...props}
  >
    {/* <Toolbar> */}
    <Typography
      variant="title"
      color="inherit"
      className={classes.title}
      id="react-admin-title"
    />
    <Logo />

    <span className={classes.spacer} />
    {/* </Typography> */}
    {/* </Toolbar> */}
  </AppBar>
));

export default CustomAppBar;
```

**Other information:**

**Environment**

* React-admin version: 2.6.0
* Last version that did not exhibit the issue (if applicable):
* React version: 16.7
* Browser: Chrome

**Comment by djhi** [comment]
If you are able to illustrate the bug or feature request with an example, please provide a sample application via one of the following means:

* CodeSandbox (https://codesandbox.io/s/349ovjmlx5)
* A sample application via GitHub

**Comment by fzaninotto** [comment]
This feature was released in 2.7.0, you need to upgrade react-admin.
